MetaMask/metamask-extension
# Replace web3js package with ethersjs package

## Explanation
We don't need to import both web3.js and ethers.
This PR replaces all imports of web3.js with ethers.



## More Information


 Fixes #15231

## Manual Testing Steps

There are two places where we use web3.js.

1. When we first load MetaMask to load balance from the blockchain.
2. When we are detecting tokens with the Refresh list button.

To test the first case, load MetaMask and make sure that the correct balance is displayed.
To test the second case, switch to Ethereum Mainnet (it works only on mainnet) and click the Refresh list button. 



## Pre-Merge Checklist


### + If there are functional changes:
